### ISSUE TITLE
[Bug] Collections accessibility updates

### ISSUE THREAD

**Issue by sblatz** [mozilla-mobile/fenix#7971]
When selecting tabs to add to a collection, each tab should be one focusable item that should read the title of the tab and then "checkbox OFF" 

Double tapping this item should select the collection.

**Comment by sblatz** [comment]
Touch base with @mcomella. I think he solved something similar in FireTV.

**Comment by sblatz** [comment]
We probably just need to make the checkbox focusable="false".
